classes/blocks/class-spreadsheet.php

```diff
@@ -113,4 +123,11 @@ public static function get_sheet( ?string $sheet_id, bool $skip_cache ): ?array
 
 		return $sheet;
 	}
+
+
+	/**
+	 * Spreadsheet has no editor style.
+	 */
+	public static function enqueue_editor_style(): void {
```

Comment:
Instead of adding this override here, wouldn't it be possible to check in the original function that the files exist? 🤔

Reply:
Yeah good point, I was thinking about the overhead of checking files on disk, but it's minimal and we serve mostly cached content, so it's easier that way. Done :+1: